cmake/FindSonic.cmake

```diff
@@ -0,0 +1,15 @@
+# This is free and unencumbered software released into the public domain.
```

Comment:
can you use standard gpl header?

Reply:
If I wanted to license the CMake module under GPL, I'd have used GPL. But it is under Unlicense: I am not copyrust.

Reply:
I do not want to argue with you about this. but don't you think it is a good idea to follow the style and license of the project you are contributing to? 😉

@Olga-Yakovleva what do you think, can we take this into the project?